reviewblitz/templates/blitz_review_approval_guidelines.html

```diff
@@ -0,0 +1,40 @@
+
+<p>Helping approve Blitz reviews is one of the logistic staff's most important duties! There are literally hundreds of reviews that enter the queue during Review Blitz, so even if you can only knock off one or two, it's a huge help to have multiple people doing approvals.</p>
+
+<p>When approving reviews in the queue, you can click on the name of the review (e.g. "Negrek's review on Morphic") to go directly to its post. A quick skim should be enough to give you a sense of whether or not a review is ready to be approved. These are the things you'll want to consider when approving reviews:</p>
+
+<ul>
+    <li>
+        <p><strong>Is the review of an acceptable quality?</strong> The baseline requirement for acceptability is that the review be 250 words or longer; the app already checks that for you. However, people do sometimes pad reviews with overlong introductions or discussions of irrelevant topics (e.g. "I like your zorua avatar, here are my thoughts on zorua"), especially when they're close to the 250-word limit. A review should primarily discuss the actual content of the fanfic it's posted on--a "review" that is mostly a complaint about how the author doesn't update enough, for example, would not qualify for Blitz points. You'll also want to check that the review is coherent and legible.</p>
+
+        <p>Also note that incomplete reviews are not eligible for Blitz--occasionally we've had people write a partial review in order to get it in by a deadline and then return later to edit the "real" full review into the thread. It isn't always clear that someone is doing this at the time the reviews is submitted, but if you become aware of someone making edits to their reviews after the fact, please let the rest of the logistics staff know!</p>
+
+    </li>
+    <li>
+        <p><strong>Is the review for an eligible story?</strong> Most threads in the fanfic forum are eligible for reviews, but there are a few exceptions. Reviews on exquisite corpses do not count, nor do reviews on fics that the author has opted out of Blitz--these are listed in the present Blitz's rules and information thread. Reviews fulfilling overdue Catnip assignments are also not eligible for Blitz points.</p>
+    </li>
+    <li>
+        <p><strong>Does the review cover at least 250 words of material?</strong> Some works may be standalone drabbles, or chaptered fics may have one or more very short chapters that don't meet the threshold necessary to qualify for Blitz points.</p>
+    </li>
+    <li>
+        <p><strong>Does the review cover the material claimed?</strong> For example, if the app indicates that the review is supposed to cover ten chapters, but it only mentions one or two events or developments in the story, the chapter count might be inflated.</p>
+    </li>
+    <li>
+        <p><strong>Did the reviewer forget to claim the chaper length bonus?</strong> The app will prevent this bonus from being awarded when it shouldn't be, but the reverse isn't true.</p>
```

Comment:
Is forgetting to claim the chapter length bonus cause for *rejecting* a review? I thought part of the point was it's optional so if you don't care you don't have to bother with the additional work?